app/models/stat.rb

```diff
@@ -64,7 +64,6 @@ def rdvs_group_by_status
       .group("status")
       .group_by_week("rdvs.starts_at", range: default_date_range, format: DEFAULT_FORMAT)
       .count
-    res = res.select { _2.positive? } # otherwise we would divide by 0 later
```

Comment:
Maybe that line messes with the order. :man_shrugging:

Reply:
yes that was the problem, maybe you missed the PR explanations below the screenshot @guillett ?!

Reply:
I read it quickly. I feel like it is working because the previous command returns ordered buckets which is fine but is a relatively strong assumption.